Python/specialize.c

```diff
@@ -637,3 +639,36 @@ _Py_Specialize_LoadGlobal(
     cache0->counter = saturating_start();
     return 0;
 }
+
+int
+_Py_Specialize_BinarySubscr(
```

Comment:
This needs stats for specialization failure types, c.f. `_Py_Specialize_LoadAttr`